src/tracker-component.js

```diff
@@ -0,0 +1,105 @@
+/* @flow */
+
+import { getClientID, getMerchantID } from '@paypal/sdk-client/src';
+
+type TrackingType = 'view' | 'cartEvent' | 'purchase';
+
+type CartEventType = 'addToCart' | 'setCart' | 'removeFromCart';
+
+type Environment = 'stage' | 'sandbox' | 'production';
+
+type Product = {|
+    id : string,
+    url : string,
+    sku? : string,
+    name? : string,
```

Comment:
title? : string instead of name
if the user has a product that doesn't have a title, the product has to have been ingested

Reply:
Done